src/QGCApplication.h

```diff
@@ -30,6 +29,8 @@
 #include "OpalLink.h"
 #endif
 
+class QQmlApplicationEngine;
```

Comment:
Can you move this down with the other forward references and remove the obvious comment about circular header include.

Reply:
This is not fixing any circular header includes, as the comment in the top of the others forward references. But Ok.